storages/multipart_round_tripper.go

```diff
@@ -58,7 +58,7 @@ func (multiPartRoundTripper *MultiPartRoundTripper) Do(request *http.Request) <-
 	if !multiPartRoundTripper.canHandleMultiUpload() {
 		log.Debugf("Multi upload for %s failed - no backends available.", request.URL.Path)
 		go func() {
-			backendResponseChannel <- BackendResponse{Response: nil, Error: errors.New("Can't handle multi upload")}
+			backendResponseChannel <- BackendResponse{Request: request, Response: nil, Error: errors.New("Can't handle multi upload")}
```

Comment:
Maybe extract `errors.New("Can't handle multi upload")` to some global, so we can later compare it in conditions?